content/pages/standards/tcs/index.md

```diff
@@ -13,23 +13,55 @@ github: https://github.com/tdwg/tcs
 Title
 : Taxonomic Concept Transfer Schema (TCS)
 
-Date created
+Permanent IRI (for citations and links)
+: <http://www.tdwg.org/standards/117>
+
+Publisher
+: [Biodiversity Information Standards (TDWG)](https://www.tdwg.org/)
+
+Ratified
 : 2005-09-16
 
 Status
-: Current (2005) standard
+: [Current (2005) standard](https://www.tdwg.org/standards/status-and-categories/)
 
 Category
-: Technical specification
-
-Permanent IRI
-: <http://www.tdwg.org/standards/117>
+: [Technical specification](https://www.tdwg.org/standards/status-and-categories/#categories%20of%20tdwg%20standards_1)
 
 Abstract
 : The TCS schema was conceived to allow the representation of taxonomic concepts as defined in published taxonomic classifications, revisions and databases. As such, it specifies the structure for XML documents to be used for the transfer of defined concepts. Valid transfer documents may either explicitly detail the defining components of taxon concepts, transfer GUIDs referring to defined taxon concepts (if and when these are available) or a mixture of the two.
 
-Creator
-: Taxonomic Names and Concepts Interest Group, [Biodiversity Information Standards (TDWG)](https://www.tdwg.org/)
-
 Bibliographic citation
-: > Taxonomic Names and Concepts Interest Group (2006) Taxonomic Concept Transfer Schema (TCS), version 1.01. Biodiversity Information Standards (TDWG) <http://www.tdwg.org/standards/117>
+: > Taxonomic Names and Concepts Interest Group. 2005. Taxonomic Concept Transfer Schema (TCS). Biodiversity Information Standards (TDWG) http://www.tdwg.org/standards/117
+
+## Maintenance group  
+TCS is maintained by the [Taxonomic Names and Concepts Interest Group](https://www.tdwg.org/community/tnc/), which has sponsored a Task Group to update TCS to make it compliant with the Standards Documentation Specification and to move it into the Current Standard category.
```

Comment:
Need confirmation from @jmacklin and @nielsklazenga that this is true prior to merging